backend/src/main/java/com/eyetrade/backend/service/CurrencyRecordService.java

```diff
@@ -51,6 +54,7 @@ public CurrencyRecord getCurrencyRecord() throws IOException {
         record.setJapanRate(rates.get(CurrencyType.JPY.toString()).doubleValue());
         record.setChinaRate(rates.get(CurrencyType.CNY.toString()).doubleValue());
         currencyRepository.save(record);
+        transactionService.checkBuySellOrder();
```

Comment:
As I understand you will check the buy orders once a day

Reply:
Yes. Because it's enough to check orders when rates are changed, we check once a day.